index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Capstone Project</title>
+    <link rel="stylesheet" href="styles.css">
+    <script defer src="script.js"></script>
+</head>
+<body>
+    <header> 
+        <ul class="nav">
+            <li><a href="#"><img src="images/facebook.png" alt="facebook logo" width="25px" height="25px"></a></li>
+            <li><a href="#"><img src="images/twitter.png" alt="twitter logo" width="25px" height="25px"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+        <div class="nav-container">
+            <a href="index.html" id="logo-link"><img src="images/logo.png" alt="uacademy logo" id="logo"></a>
+            <ul class="navbar">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a class="campaign" href="#">CC Campaign</a></li>
+            </ul>
+        </div>
+        <a href="#" class="toggle-btn">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </a>
+        <div id="hamburger-menu">
+            <span class="close-btn">&times;</span>
+            <ul class="hamburger-menu">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+      </div>
+    </header>
+    <section id="home">
+        <h3>"Hello! Sharing world"</h3>
+        <h1>Udemy Online Courses 2023</h1>
+        <p class="course-description">One of the best web development course believing in the value of improvement via learning, creating new opportunities for people from over 180 countries is taking place in February</p>
+        <div class="time">2023.02.6(MON)~7(TUS)</div>
+        <p class="course-location">@San Fransisco, Web development and more</p>
+    </section>
+    <section id="about">
+        <div class="main-program">
+            <h4>Main Program</h4>
+            <div class="underline"></div>
+            <div class="flex-container">
+                <div class="programs">
```

Comment:
Kindly use semantic article element instead of div for each card for making your web page more SEO friendly.